src/App.tsx

```diff
@@ -1,25 +1,52 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { getItemsOnPage } from './utils/pagination/getItemsOnPage';
+import { getItemsOnPageCount } from './utils/pagination/getItemsOnPageCount';
+import { Pagination } from './components/Pagination';
+import { EPaginationSize } from './enums/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState<EPaginationSize>(
+    EPaginationSize.Medium,
+  );
+
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const perPageSizeHandler = (e: ChangeEvent<HTMLSelectElement>) => {
```

Comment:
```suggestion
  const handlePerPageChange = (event: ChangeEvent<HTMLSelectElement>) => {
```
- Avoid one-letter variable names
- handler name should start from `handle` and if it's a prop from `on` https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364